src/main/java/com/techsophy/tsf/runtime/form/controller/impl/FormAclControllerImpl.java

```diff
@@ -39,11 +42,13 @@ public ApiResponse<FormAclDto> getFormAcl(String formId) throws JsonProcessingEx
     }
 
     @Override
+    @Transactional
```

Comment:
we dont need this

Reply:
ok vaibhav removing transaction on get api